Derikka/EmbeddedSingleCavity.ui

```diff
@@ -0,0 +1,246 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
So we wouldn't actually be using the layouts for anything cosmetic, this would be strictly make our coding easier with findChildren()?

Reply:
Correct!

Reply:
For our purposes, is there a difference between just right clicking and adjusting the form layout vs including an actual QHBoxLayout object?

Reply:
Changing the layout works! With this method, it's not immediately obvious in designer that a layout is being used at all, but findChildren(QHBoxLayout) picks it up no problem!

Reply:
So in the end, I made the EmbeddedSingleCavity.ui have a horizontal form layout, and the cryomodule.ui has a vertical form layout. The cavityCud.ui on the other hand does have an actual QHBoxLayout and its form layout is set in a grid. I may have to add more QHBoxLayouts or QVBoxLayouts into the final cavityCud.ui in order to add linac labels

Reply:
Oh hmm. I think we need to think this through a little bit then, because what I'm hoping is that whatever layouts we choose for cryomodules and cavities are unique, because otherwise findChildren is gonna return a crap ton of stuff that we'd end up needing to filter (kind of defeating the purpose)